### DISCUSSION TITLE
Conflict with  Version ^8.0 requires illuminate/console ^9.0 -> found illuminate/console[v9.0.0-beta.1, 9.x-dev]. How can i fix it?

### DISCUSSION THREAD

**Discussion by wemersonrv** [spatie/laravel-backup#1447] [Q&A] [answered]
Hello guys,

I am trying to install the package on a Laravel 8.x app, that was recently upgraded from a previous Laravel version. and it gives me the conflict below:

```bash
 Problem 1
    - Root composer.json requires spatie/laravel-backup ^8.0 -> satisfiable by spatie/laravel-backup[8.0.0].
    - spatie/laravel-backup 8.0.0 requires illuminate/console ^9.0 -> found illuminate/console[v9.0.0-beta.1, 9.x-dev] but these were not loaded, likely because it conflicts with another require.
```

How can i fix it?

here my dependencies:
```json
{
    "require": {
        "php": "^8.0",
        "fideloper/proxy": "^4.4",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.4",
        "laravel/framework": "^8.0",
        "laravel/helpers": "^1.5",
        "laravel/nova": "^3.30",
        "laravel/tinker": "^2.7"
    },
    "require-dev": {
        "facade/ignition": "^2.5",
        "fzaninotto/faker": "^1.9.1",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^5.10",
        "phpunit/phpunit": "^9.5.10"
    },
}
```

**Comment by erikn69** [comment, marked as answer]
`spatie/laravel-backup ^8.0` only supports laravel 9 
https://github.com/spatie/laravel-backup/blob/20ac14e881bb1826bbd7fb30a7d9051d9cbfe78e/composer.json#L23-L28
For laravel 8 use `spatie/laravel-backup ^7.0`

**Reply by wemersonrv** [reply]
> `spatie/laravel-backup ^8.0` only supports laravel 9
> 
> https://github.com/spatie/laravel-backup/blob/20ac14e881bb1826bbd7fb30a7d9051d9cbfe78e/composer.json#L23-L28
> 
> 
> For laravel 8 use `spatie/laravel-backup ^7.0`

I see... well when i try to install via `composer require spatie/laravel-backup` in a Laravel 8 project, it try to install the `^8` thats the issue. Something is forcing composer to use the wrong version... Or do i need to manually force the version when install?

**Reply by erikn69** [reply]
Manually force
